app/controllers/sessions_controller.rb

```diff
@@ -1,40 +1,36 @@
 class SessionsController < ApplicationController
   before_action :check_logged_out_user, only: [:new, :create]
- 
+
   def new
   end
 
   def create
-    user = User.find_by(username: params[:session][:username].downcase)
-
-    if user && user.authenticate(params[:session][:password])
-      # Log in and redirect to the user profile page
-      if user.email_confirmed
-        if user_approved(user)
```

Comment:
Will be a tiny merge conflict here, but it's nbd; I'll accept your changes over mine